### PR TITLE
Structure stage 0 better.

### PR DESCRIPTION
**Pull request by jonathanrocher**
Better structure for the script so it's easier to trait-ify it. Also simplified its name.

### CODE REVIEW

File: stage0_starting_script/face_detect.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import patches
+
+# Select image file -----------------------------------------------------------
+
+image_path = join("..", "sample_images", "IMG-0311_xmas_2020.JPG")
+image_path2 = join("..", "sample_images", "owls.jpg")
+
+for path in [image_path, image_path2]:
```
Comment (siddhantwahal) [inline stage0_starting_script/face_detect.py:19]:
```suggestion
image_paths = [
    join("..", "sample_images", "IMG-0311_xmas_2020.JPG"),
    join("..", "sample_images", "owls.jpg")
]

for path in image_paths:
```

File: stage0_starting_script/face_detect.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    img_metadata = {TAGS[k]: v for k, v in img._getexif().items() if k in TAGS}
+
+    # Detect faces ------------------------------------------------------------
+
+    # Load the trained file from the module root.
+    trained_file = data.lbp_frontal_face_cascade_filename()
+
+    # Initialize the detector cascade.
+    detector = Cascade(trained_file)
```
Comment (siddhantwahal) [inline stage0_starting_script/face_detect.py:30]: These could be outside the for loop.

File: stage0_starting_script/face_detect.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                                           max_size=(600, 600))
+
+    img_metadata["Number of faces detected"] = len(detected)
+    print(img_metadata)
+
+    # Visualize results -------------------------------------------------------
+
+    plt.imshow(img)
+    img_desc = plt.gca()
+    plt.set_cmap('gray')
```
Comment (siddhantwahal) [inline stage0_starting_script/face_detect.py:45]:
```suggestion
```
Do we really need to set the colormap?

**Review by siddhantwahal — APPROVED** [review]
A few optional suggestions:

File: stage0_starting_script/face_detect.py (hunk 2 above)
Comment (jonathanrocher) [reply stage0_starting_script/face_detect.py]: Done.

File: stage0_starting_script/face_detect.py (hunk 3 above)
Comment (jonathanrocher) [reply stage0_starting_script/face_detect.py]: Removed.

**Comment by jonathanrocher** [comment]
Thanks @siddhantwahal for all the suggestions, all done.

**Pull request merged**